index.html

```diff
@@ -8,13 +8,15 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
     <script src="https://kit.fontawesome.com/594a9a3356.js" crossorigin="anonymous"></script>
+    <script src="index.js"></script>
     <title>Practise</title>
 </head>
 <body>
     <header>
         <nav class="navbar">
+            <span class="navbar__logo"><a class="navbar__link" href="#"><i class="fa-brands fa-uikit"></i></a></span>
```

Comment:
do you need both span and anchor here?

Reply:
After review - span is not needed. Anchor alone handles all the expected layout and behaviour. I will get rid of it